server/src/weaverbird/backends/pypika_translator/translators/snowflake.py

```diff
@@ -15,6 +15,22 @@
 Self = TypeVar("Self", bound="SQLTranslator")
 
 
+class SnowflakeQuery(Query):
+    @classmethod
+    def _builder(cls, **kwargs: Any) -> "SnowflakeQueryBuilder":
+        return SnowflakeQueryBuilder(**kwargs)
+
+
+class SnowflakeQueryBuilder(QueryBuilder):
+    QUOTE_CHAR = '"'
+    ALIAS_QUOTE_CHAR = '"'
+    QUERY_ALIAS_QUOTE_CHAR = ""
+    QUERY_CLS = SnowflakeQuery
+
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(dialect=Dialects.SNOWFLAKE, **kwargs)
+
+
```

Comment:
```suggestion
from pypika.dialects import SnowflakeQueryBuilder

class QuotedSnowflakeQueryBuilder(SnowflakeQueryBuilder):
    QUOTE_CHAR = '"'
```

We want to be as close as possible to the PyPika implem

Reply:
https://github.com/ToucanToco/weaverbird/pull/1533/commits/cf2ee8780ca551454e5d1212304757a9b1996bbf